autocomplete/complete.fish

```diff
@@ -25,7 +25,8 @@ if test -d "$HOME/.tldrc/tldr/pages"
         set cmpl $cmpl (__tldr_get_files sunos)
     end
 
-    complete -c tldr -a (echo $cmpl | sort | uniq)
+    complete -c tldr -f -a (echo $cmpl | sort | uniq) -n "__fish_use_subcommand"
```

Comment:
Your command `complete -c tldr -f -n "__fish_use_subcommand" -a (echo -ns $cmpl\n | sort | uniq)` mostly works, actually. In fish, to make subshells work with newlines, you have to use `"$(...)"` syntax instead of `()`.

So this works without error:
```fish
complete -c tldr -f -n "__fish_use_subcommand" -a "$(echo -ns $cmpl\n | sort | uniq)"
```

Reply:
However, from my tests, `fish` will automatically filter out duplicate values **and** sort the suggestions of completions, so your simpler suggestion of `complete -c tldr -f -n __fish_use_subcommand -a $cmpl` probably works just the same.